src/main/java/pl/noip/lolstats/lol/stats/Exceptions/NoNameException.java

```diff
@@ -0,0 +1,4 @@
+package pl.noip.lolstats.lol.stats.Exceptions;
+
+public class NoNameException extends RuntimeException {
```

Comment:
Rename to something like no name in token or create base exception class, for example JwtException and extend it.